docs/CONTAINERS.md

```diff
@@ -2,8 +2,9 @@
 
 Subsets of all containers can be downloaded:
 
-- For processing, germline and somatic variant calling and Reports:
+- For processing, germline variant calling and Reports:
```

Comment:
It's not used for somatic VC?

Reply:
yes, that's why I said **add** r-base and runallelecount for somatic VC

Reply:
True, I missed that. But for example FreeBayes still use the sarek container right?

Reply:
Ah, ok I see now! Never mind this.